loops_lab.js

```diff
@@ -0,0 +1,195 @@
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Overall good demonstration and implementation of for and while loops. Other than missing variable declaration inside for loops, everything else looks fine. Good job and keep up the good work!